docs/api-reference/layer.md

```diff
@@ -167,7 +167,7 @@ Requires `pickable` to be true.
 
 * Default: `[0, 0, 128, 128]`
 
-RGBA color to be used to render highlighted object.
+RGBA color to be used to render highlighted object. When 3 component (RGB) array is specified, a default value of 128 is used for alpha.
```

Comment:
We should use `255` for consistency with the rest of the layer APIs.
This won't break any existing use case as 3-component value doesn't work right now.